source/_components/nissan_leaf.markdown

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: "Nissan Leaf"
+description: "Instructions for how to integrate Nissan Leaf(s) into Home Assistant."
+date: 2019-01-05 07:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nissan.png
+ha_category: Car
+ha_release: 0.88
+ha_iot_class: "Cloud Polling"
+redirect_from:
+ - /components/binary_sensor.nissan_eaf/
+ - /components/sensor.nissan_leaf/
+ - /components/switch.nissan_leaf/
+ - /components/device_tracker.nissan_leaf/
+---
+
+The `nissan_leaf` component offers integration with the 
+[NissanConnect EV](http://youplus.nissan.co.uk/GB/en/YouPlus/ConnectedServices.html) cloud 
+service. NissanConnect EV was previously known as Nissan Carwings. It offers:
+
+ * sensors for the battery status and range
+ * a switch to start and stop the climate control  
+ * a switch to start the car charging (cannot be stopped remotely - API limitation)
+ * a device tracker to locate the car (only on later Leaf models)
+
+## {% linkable_title Configuration %}
+
+To use Nissan Leaf in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+nissan_leaf:
+  username: "YOUR_USERNAME"
+  password: "YOUR_PASSWORD"
+  region: "YOUR_REGION"
+```
+
+A more advanced example for setting the update interval:
+
+```yaml
+# Example configuration.yaml entry
+nissan_leaf:
+  username: "john.smith@somewhere.com"
+  password: "mysecurepassword"
```

Comment:
```suggestion
  password: YOUR PASSWORD
```

Reply:
Thanks.  Made change but with added quotation marks in commit https://github.com/home-assistant/home-assistant.io/pull/8073/commits/56c5dd93bbcdc4e6bceebc9b1828c6a1671ea641